synapse/util/caches/lrucache.py

```diff
@@ -341,7 +341,7 @@ def __init__(
         # Default `clock` to something sensible. Note that we rename it to
         # `real_clock` so that mypy doesn't think its still `Optional`.
         if clock is None:
-            real_clock = Clock(reactor)
+            real_clock = Clock(reactor)  # type: ignore[arg-type]
```

Comment:
Why does this complain?

Reply:
the import type is `module`, not `IReactorTime`. I suppose this might be better as a `cast`, so I'll do that.